package.json

```diff
@@ -137,6 +137,7 @@
     "eslint-config-prettier": "6.11.0",
     "eslint-plugin-jsdoc": "28.6.1",
     "eslint-plugin-prettier": "3.1.4",
+    "eslint-plugin-react": "7.21.5",
```

Comment:
This dependency is to be removed after we kick in the updated grafana-esint-config